.travis.yml

```diff
@@ -1,24 +1,12 @@
-dist: xenial
+os: linux
+dist: bionic
```

Comment:
What's the reason for changing  dist from  `xenial` to `bionic`?

Reply:
I guess she just merged [the original repo](https://github.com/fanduel-oss/jsonrpc2-elixir/tree/master) to our fork so none of the changes are made by us